item.rb

```diff
@@ -0,0 +1,25 @@
+# item classs
+class Item
+  def initialize(date)
+    @id = Random.rand(1..1000)
+    @genre = nil
+    @author = nil
+    @label = nil
+    @publish_date = date
+    @archived = move_to_archive(@publish_date)
```

Comment:
I think this variable should hold an array, is that correct?

Reply:
Hi, @jssol. Relationships between (Genre, Author, Label) and Item are one-to-many relationships which means:
- A Genre has many Items.
-  A Label has many Items.
- A Author has many Items.
- an Item has one Label, Author, and Genre

Reply:
Hi, @jssol. Relationships between (Genre, Author, Label) and Item are one-to-many relationships which means:
- A Genre has many Items.
-  A Label has many Items.
- A Author has many Items.
- an Item has one Label, Author, and Genre